### PR TITLE
Woo-Connect Insights: update and fix non-day units for products, categories and coupon pages

### LINKED ISSUE #15760: Woo-Connect Insights: Fix non-day periods in Product, Category and Coupon pages
These pages currently don't support week, month and year periods.

### PR DESCRIPTION
**Pull request by greenafrican**
Fixes #15760

### CODE REVIEW

File: client/extensions/woocommerce/app/store-stats/constants.js [hunk 1]
```diff
@@ -53,23 +53,23 @@ export const topProducts = {
...
 export const topCoupons = {
 	basePath: '/store/stats/coupons',
-	title: translate( 'Coupons' ),
+	title: translate( 'Most Used Coupons' ),
 	values: [
 		{ key: 'name', title: translate( 'Title' ) },
-		{ key: 'quantity', title: translate( 'Quantity' ) },
-		{ key: 'total', title: translate( 'Total' ) },
+		{ key: 'quantity', title: translate( 'Used' ) },
+		{ key: 'total', title: translate( 'Total Discount' ) },
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/store-stats/constants.js]: Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Discount' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wcss-non-day-listviews)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/store-stats/constants.js [hunk 2]
```diff
@@ -53,23 +53,23 @@ export const topProducts = {
 
 export const topCategories = {
 	basePath: '/store/stats/categories',
-	title: translate( 'Categories' ),
+	title: translate( 'Top Categories' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/store-stats/constants.js]: Hi! I've found a possible matching string that has already been translated **86** times:
`translate( 'Categories: ' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wcss-non-day-listviews)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by bluefuton** [comment]
Sometimes we get the appearance of duplicates in the list - @greenafrican reckons these are product variations:

screen shot 2017-07-06 at 11 46 36

Would be great combine those or list the product variation in the table.

**Comment by greenafrican** [comment]
Thanks @bluefuton going to track that here - https://github.com/Automattic/wp-calypso/issues/15910

**Pull request merged**